packages/bpk-component-segmented-control/src/BpkSegmentedControl.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ReactNode} from 'react';
+import { useState } from 'react';
+
+import { cssModules } from '../../bpk-react-utils';
+
+import SEGMENT_TYPES from './segmentTypes';
+
+import type { SegmentTypes } from './segmentTypes';
+
+import STYLES from './BpkSegmentedControl.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export type Props = {
+  buttonContents: string[] |  ReactNode[];
+  type?: SegmentTypes;
+  onItemClick: (id: number) => void; // index param to track which one is clicked
```

Comment:
If you add a comment on this above this line to be 

```
/*
* Index parameter to track which is clicked
*/
```

This will then show up in our prop tables